Parking/motorcylce_parking.csv

```diff
@@ -0,0 +1,9 @@
+name,description,note,latitude,longitude

```

Comment:
:key:  note ... I believe it's called *motorcycle_parking* not motorcylce_parking"